databricks/koalas/internal.py

```diff
@@ -638,6 +642,15 @@ def spark_df(self) -> spark.DataFrame:
     def pandas_df(self):
         """ Return as pandas DataFrame. """
         sdf = self.spark_internal_df
+
+        if LooseVersion(pyarrow.__version__) >= LooseVersion("0.15") and \
+                LooseVersion(pyspark.__version__) < LooseVersion("3.0"):
+            # There seems a bug in PyArrow 0.15 with PySpark 2.4 and below.
+            # TODO(Hyukjin): I have no idea why only inserting limit works.
+            #   Other APIs like noop map, explicitly shuffle and sample with
+            #   fraction 1 still don't work.
+            sdf = sdf.limit(2147483647)
```

Comment:
I have no idea at all why only `limit` works. I tried some other APIs I could think like map with empty function, sort, sample with fraction 1.0 but none of them work.